Aptoide/aptoide-client-v8
# [ASV-1267] Promotions events (reload and icon impression)

**What does this PR do?**

 Adds 2 events related to promotions
-When you’re in the captcha dialog and you press reload an event with action reload should be sent
-When you go to home, if there are promotions an event with action impression should be sent
(Check ticket below for more detail on ticket information)

**Database changed?**

No

**Where should the reviewer start?**

HomeAnalytics.java
PromotionAnalytics.java
ClaimPromotionDialogPresenter.java

**How should this be manually tested?**

Go to home and check if the impression event is sent on the logicat
Go to the promotions fragment and install the wallet and go trough the claim flow. When you reach the captcha, press reload and check if the event appears on the logicat

**What are the relevant tickets?**

https://aptoide.atlassian.net/browse/ASV-1267

**Questions:**

   Does this add new dependencies which need to be added? (Eg. new libs, new keys, etc.) 




**Code Review Checklist**
